diff -Naur /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/Testbenches/ftp_test/Sources/Navdata/navdata.c /root/ARDrone_SDK_2_0_1/Examples/Linux/Testbenches/ftp_test/Sources/Navdata/navdata.c
--- /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/Testbenches/ftp_test/Sources/Navdata/navdata.c	2011-04-06 11:23:05.000000000 +0000
+++ /root/ARDrone_SDK_2_0_1/Examples/Linux/Testbenches/ftp_test/Sources/Navdata/navdata.c	2017-03-26 20:56:20.000000000 +0000
@@ -8,7 +8,7 @@
 #include "control_states.h"
 
 /* Initialization local variables before event loop  */
-inline C_RESULT demo_navdata_client_init( void* data )
+C_RESULT demo_navdata_client_init( void* data )
 {
   return C_OK;
 }
@@ -50,7 +50,7 @@
 }
 
 /* Receving navdata during the event loop */
-inline C_RESULT demo_navdata_client_process( const navdata_unpacked_t* const navdata )
+C_RESULT demo_navdata_client_process( const navdata_unpacked_t* const navdata )
 {
 	static int cpt=0;
 
@@ -106,7 +106,7 @@
 }
 
 /* Relinquish the local resources after the event loop exit */
-inline C_RESULT demo_navdata_client_release( void )
+C_RESULT demo_navdata_client_release( void )
 {
   return C_OK;
 }
diff -Naur /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/sdk_demo/Sources/Navdata/navdata.c /root/ARDrone_SDK_2_0_1/Examples/Linux/sdk_demo/Sources/Navdata/navdata.c
--- /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/sdk_demo/Sources/Navdata/navdata.c	2010-12-22 13:26:40.000000000 +0000
+++ /root/ARDrone_SDK_2_0_1/Examples/Linux/sdk_demo/Sources/Navdata/navdata.c	2017-03-26 20:55:20.000000000 +0000
@@ -4,13 +4,13 @@
 #include <stdio.h>
 
 /* Initialization local variables before event loop  */
-inline C_RESULT demo_navdata_client_init( void* data )
+C_RESULT demo_navdata_client_init( void* data )
 {
   return C_OK;
 }
 
 /* Receving navdata during the event loop */
-inline C_RESULT demo_navdata_client_process( const navdata_unpacked_t* const navdata )
+C_RESULT demo_navdata_client_process( const navdata_unpacked_t* const navdata )
 {
 	const navdata_demo_t*nd = &navdata->navdata_demo;
 
@@ -28,7 +28,7 @@
 }
 
 /* Relinquish the local resources after the event loop exit */
-inline C_RESULT demo_navdata_client_release( void )
+C_RESULT demo_navdata_client_release( void )
 {
   return C_OK;
 }
