diff -Naur /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/Navigation/Build/Makefile /root/ARDrone_SDK_2_0_1/Examples/Linux/Navigation/Build/Makefile
--- /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/Navigation/Build/Makefile	2017-03-23 13:37:15.756488190 +0000
+++ /root/ARDrone_SDK_2_0_1/Examples/Linux/Navigation/Build/Makefile	2017-03-23 12:49:33.116567737 +0000
@@ -128,7 +128,7 @@
 
 GENERIC_LIB_PATHS=-L$(GENERIC_TARGET_BINARIES_DIR)
 GENERIC_CFLAGS+=-DLIBUDEV_I_KNOW_THE_API_IS_SUBJECT_TO_CHANGE
-GENERIC_LIBS+=-liw -lpc_ardrone -lgthread-2.0 -lgtk-x11-2.0 -lrt -lxml2 -ludev -lswscale -lSDL
+GENERIC_LIBS+=-liw -lpc_ardrone -lgthread-2.0 -lgtk-x11-2.0 -lrt -lxml2 -ludev -lswscale -lSDL -lm
 
 #ifeq ($(USE_ARDRONE_VISION),yes)
 #   GENERIC_LIBS+=-lpc_vision
diff -Naur /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/Testbenches/ftp_test/Sources/ardrone_testing_tool.c /root/ARDrone_SDK_2_0_1/Examples/Linux/Testbenches/ftp_test/Sources/ardrone_testing_tool.c
--- /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/Testbenches/ftp_test/Sources/ardrone_testing_tool.c	2017-03-23 13:37:46.953596506 +0000
+++ /root/ARDrone_SDK_2_0_1/Examples/Linux/Testbenches/ftp_test/Sources/ardrone_testing_tool.c	2017-03-23 13:02:56.684913483 +0000
@@ -282,8 +282,8 @@
 
   int runOperation = 1;
 
-  uint64_t totalOp = 0;
-  uint64_t opOk = 0;
+  unsigned long long totalOp = 0;
+  unsigned long long opOk = 0;
 
   
   title ("\n\n------ CLEANING ------\n\n");
diff -Naur /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/video_demo/Build/Makefile /root/ARDrone_SDK_2_0_1/Examples/Linux/video_demo/Build/Makefile
--- /root/ARDrone_SDK_2_0_1-orig/Examples/Linux/video_demo/Build/Makefile	2017-03-23 13:37:33.217108489 +0000
+++ /root/ARDrone_SDK_2_0_1/Examples/Linux/video_demo/Build/Makefile	2017-03-23 12:59:05.420697760 +0000
@@ -44,7 +44,7 @@
 GENERIC_INCLUDES:=$(addprefix -I,$(GENERIC_INCLUDES))
 
 GENERIC_LIB_PATHS=-L$(GENERIC_TARGET_BINARIES_DIR)
-GENERIC_LIBS=-lpc_ardrone -lrt -lgtk-x11-2.0 -lcairo
+GENERIC_LIBS=-lpc_ardrone -lrt -lgtk-x11-2.0 -lcairo -lgobject-2.0 -lgdk-x11-2.0 -lm
 
 
 SDK_FLAGS+="USE_APP=yes"
